homeassistant/components/lm75/config_flow.py

```diff
@@ -0,0 +1,33 @@
+"""Config flow for LM75."""
+import voluptuous as vol
+
+from homeassistant import config_entries
+
+from . import DOMAIN
+
+
+class SimpleConfigFlow(config_entries.ConfigFlow, domain=DOMAIN):
+    """Handle a config flow for local_ip."""
+
+    VERSION = 1
+
+    async def async_step_user(self, user_input=None):
+        """Handle the initial step."""
+        if user_input is not None:
+            if any(
+                user_input["name"] == entry.data["name"]
+                for entry in self._async_current_entries()
+            ):
+                return self.async_abort(reason="already_configured")
+
+            return self.async_create_entry(title=user_input["name"], data=user_input)
```

Comment:
Can we connect to the device before creating the entry to check that the connection works?